share/spice/cricket/cricket.css

```diff
@@ -0,0 +1,136 @@
+.zci--cricket .card {
```

Comment:
The tile design should stay as we have it -- no box shadows etc. That's all Google design, not DDG 

https://duckduckgo.com/?q=blue+jays&iai=8764&ia=mlb